staging/src/k8s.io/sample-controller/README.md

```diff
@@ -94,6 +90,10 @@ kubectl create -f artifacts/examples/example-foo.yaml
 
 # check deployments created through the custom resource
 kubectl get deployments
+
+# assumes you have a working kubeconfig, not required if operating in-cluster
+go build -o sample-controller .
+./sample-controller -kubeconfig=$HOME/.kube/config
```

Comment:
It seems like the ‘get deployments’ step should happen after we run the controller, else the user won’t see the Deployment resource created when they run it 😊

Reply:
`kubectl get deployments` should check if `example-foo.yaml` is deployed successfully, right?